templates/config.edn

```diff
@@ -1,137 +1,160 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ :preferred-format "Markdown"
+
+ ;; Set the preferred workflow style.
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
  :preferred-workflow :now
 
- ;; The app will ignore those directories or files.
- ;; E.g. :hidden ["/archived" "/test.md" "../assets/archived"]
+ ;; Exclude directories/files.
+ ;; Example usage: 
+ ;; :hidden ["/archived" "/test.md" "../assets/archived"]
  :hidden []
 
- ;; When creating the new journal page, the app will use your template if there is one.
- ;; You only need to input your template name here.
- :default-templates
- {:journals ""}
-
- ;; Set a custom date format for journal page title
- ;; Example:
- ;; :journal/page-title-format "EEE, do MMM yyyy"
+ ;; Define the default journal page template.
+ ;; Enter the template name between the quotes.
+ :default-templates {:journals ""}
+
+ ;; Set a custom date format for the journal page title.
+ ;; Default value: "MMM do, yyyy"
+ :journal/page-title-format "MMM do, yyyy"
+
+ ;; Specify the journal filename format.
+ ;; !Warning:
+ ;;   - Changing this setting may cause issues if not configured correctly.
+ ;;     This configuration option is not retroactive and
+ ;;     will only affect newly created journals.
+ ;; You may use any valid date format string.
+ ;; Default value: "yyyy-MM-dd"
+ ;; :journal/file-name-format "yyyy_MM_dd"
 
- ;; Whether to enable hover on tooltip preview feature
- ;; Default is true, you can also toggle this via setting page
+ ;; Enable tooltip preview on hover.
+ ;; Default value: true
  :ui/enable-tooltip? true
 
- ;; Show brackets around page references
- ;; :ui/show-brackets? true
+ ;; Display brackets around [[page]] references.
+ ;; Default value: true
+ :ui/show-brackets? true
 
- ;; Enable showing the body of blocks when referencing them.
+ ;; Display the complete block content when referencing.
+ ;; Default value: false
  :ui/show-full-blocks? false
 
- ;; Expand block references automatically when zoom-in
+ ;; Automatically expand block references when zooming in.
+ ;; Default value: true
  :ui/auto-expand-block-refs? true
 
- ;; Enable Block timestamp
+ ;; Enable Block timestamps.
+ ;; Default value: false
  :feature/enable-block-timestamps? false
 
- ;; Enable remove accents when searching.
- ;; After toggle this option, please remember to rebuild your search index by press (cmd+c cmd+s).
+ ;; Disable accent marks when searching.
+ ;; After changing this setting, rebuild the search index by pressing (^C ^S).
```

Comment:
Should this shortcut produce any kind of visual or console response? If so, this shortcut doesn't work for me on a mac. If not, let's add a note saying as much here.

Reply:
Ah, it's `Cmd-C Cmd-S` on macOS. Weird shortcut, but sure. We should list both in the comment here.

BTW, this produces visual output when it finishes but not when it starts. But it's fast, so I guess that's fine.

Reply:
I see further down we specify in-app shortcut syntax as being like `ctrl+c ctrl+s`; I think we should use that here. And we should do it for both the windows/linux and mac shortcuts.